joplin/pages/official_documents_page/factories.py

```diff
@@ -28,7 +28,7 @@ class Meta:
         model = OfficialDocumentCollectionDocument
 
 
-class OfficialDocumentPageFactory(JanisBasePageFactory):
+class OfficialDocumentPageFactory(JanisBasePageWithTopicsFactory):
```

Comment:
OfficialDocumentPage inherits from JanisBasePage, that's why the factory is JanisBasePageFactory
`OfficialDocumentPage(JanisBasePage)` 

otherwise officialdocumentpages would need topics associated with them